### DISCUSSION TITLE
How to install RHVoices on Linux?

### DISCUSSION THREAD

**Discussion by DGDays** [RHVoice/RHVoice#265] [Q&A]
Good day!  Met with a problem installing voices on Linux.  I used to work with the Russian set of voices on Windows and everything was fine, but on Linux I do not know how to properly install the voices so that python packages can see them.  Help me please

**Comment by DGDays** [comment]
Yes, by the way, he sees the voices, but he separates them only by languages ​​that do not work, and as such he does not notice voices

**Comment by alex19EP** [comment]
pleas provide more info:

1. what distribution are you using?
2. how do you use rhvoice with python on windows?

**Reply by DGDays** [reply]
I use kali linux (the same debian)
I work with RHVoice on windows via the python module pyttsx3

**Reply by alex19EP** [reply]
as i understood pyttsx3 supports only espeak on linux. so driving rhvoice with it isn't possible.

**Reply by DGDays** [reply]
It's sad. I thought that somehow I did not install the voices that way ( And he just doesn't want to work with him...
